.github/workflows/nightly-acceptance-tests.yml

```diff
@@ -0,0 +1,60 @@
+name: Nightly Acceptance Tests
```

Comment:
If possible I'd prefer to stick to one mechanism for CI. GH actions (reportedly) run into issues when running multiple k3d clusters due to the small/limited machine sizes. The buildkite machines are much beefier.

Reply:
Hmmm, I see what you're doing with the build matrix and I'm not sure how to easily set that up in buildkite, especially with passing build artifacts around. 

I think we need to draw a hard line and pick one or the other. Maintaining both is going to be a nightmare long term as we're already fighting to keep up with buildkite/task as is.